vsdownload.py

```diff
@@ -167,15 +168,26 @@ def setPackageSelection(args, packages):
         args.package = defaultPackages
 
     if args.sdk_version != None:
+        found = False
+        versions = []
         for key in packages:
             if key.startswith("win10sdk") or key.startswith("win11sdk"):
                 base = key[0:8]
+                version = key[9:]
+                if re.match(r'\d+\.\d+\.\d+', version):
+                    versions += [version]
                 sdkname = base + "_" + args.sdk_version
                 if key == sdkname:
+                    found = True
                     args.package.append(key)
                 else:
                     args.ignore.append(key)
-        p = packages[key][0]
+        if not found:
+            print("WinSDK version " + args.sdk_version + " not found")
+            print("Available versions:")
+            for v in versions:
+                print("    " + v)
```

Comment:
It could be better to sort `versions` explicitly print it out.

Reply:
Ok, done, thanks!

(It could perhaps also be nice to have a way to get these listed without having to mistype a version number first, but I'm not sure if that's worth the extra effort, doing e.g. `--sdk-version list` works just fine in this way too, even if the tool tells you it was an error.)